clash-ghc/src-bin-861/Clash/Main.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE CPP, NondecreasingIndentation, ScopedTypeVariables, TupleSections #-}
```

Comment:
ScopedTypeVariables and TupleSections are default now here, no?

Side note: I really dislike multiple extensions in one pragma for this reason, they're harder to spot later

Reply:
Yeah my regex didn't catch it. I'll split it out across multiple lines.